Description: patch for log path in log4j.xml
 This patch fix the log path issue in log4j.xml.
 .
 gluu-oxd-server (3.0.7-SNAPSHOT-9) trusty; urgency=low
 .
   * Initial release (Closes: #1232)
Author: Adrian Alves <adrian@gluu.org>
Bug-Debian: http://bugs.debian.org/1232

Origin: gluu, 
Forwarded: not-needed

--- gluu-oxd-server-3.0.7-SNAPSHOT.orig/oxd-server/src/main/resources/log4j.xml
+++ gluu-oxd-server-3.0.7-SNAPSHOT/oxd-server/src/main/resources/log4j.xml
@@ -10,7 +10,7 @@
     </appender>
 
     <appender name="FILE" class="org.apache.log4j.DailyRollingFileAppender">
-        <param name="File" value="oxd-server.log"/>
+        <param name="File" value="/var/log/oxd-server.log"/>
         <param name="DatePattern" value="'.'yyyy-MM-dd"/>
         <param name="BufferSize" value="1000"/>
         <layout class="org.apache.log4j.PatternLayout">
